### DISCUSSION TITLE
@media (min-width: 1280px) to @media (min-width: 1281px)

### DISCUSSION THREAD

**Discussion by Mohit1310** [tailwindlabs/tailwindcss#13565] [Help]
In the below 2 image's there are 3 div's main upper part div and table div, I have added a container class for main div and table div but I am getting a problem when I go to screen size from 1536px width to 1280px  my mx-auto closes in but my second image which has a screen width of 1280px my table has become full screen and my mx-auto feels like it is not applied so how can it be solved right now I have just changed it to the below code for default container class.
```
@media (min-width: 1281px) {
  .container {
    max-width: 1280px;
  }
}
```

![image](https://github.com/tailwindlabs/tailwindcss/assets/70746260/18649806-e009-4886-98c6-206e49c10a97)

![image](https://github.com/tailwindlabs/tailwindcss/assets/70746260/84aab9e8-5257-473d-9fc8-dfcdb66c3b04)

**Comment by wongjn** [comment]
Please consider providing a [Tailwind Play](https://play.tailwindcss.com) that reproduces the issue you are having.

**Reply by Mohit1310** [reply]
[Tailwind Play](https://play.tailwindcss.com/7K75qayMgA) link

**Reply by alex-krasikau** [reply]
Hey @Mohit1310!

Thank you for sharing the link.

To address this issue, a common approach is to add horizontal padding for a safe zone when content reaches the screen edges: `container mx-auto px-12`.

I added an additional container to your example to demonstrate how it works - https://play.tailwindcss.com/BvCwd34xa4.
